day_6/questions.md

```diff
@@ -2,12 +2,36 @@
 
 1. In your own words, what is a Class?
 
+  - a syntax template to create objects and values along with attributes and methods.
+
 1. In relation to a Class, what is a property?
 
+  - A property acts like an attribute of the class.
```

Comment:
Hmm I agree, but feel like that's a synonym. Can you dig in more to the definition? What if an incoming M0 student asked you to describe it, what would you say?